src/controllers/likings.controller.js

```diff
@@ -10,6 +10,15 @@ const LikingsController = {
     } catch (error) {
       return sendResult(res, 400, 'Wrong accommodation ID sent');
     }
+  },
+
+  async getAccommdationLikes(req, res) {
```

Comment:
Great job so far! however, it would be great if you can also provide number of likes when someone is getting single accommodations.

Reply:
Thank you Lemoisson, this is for a single accommodation